packages/styled-components/src/constructors/test/keyframes.test.js

```diff
@@ -36,7 +37,7 @@ describe('keyframes', () => {
         opacity: 1;
       }
     `.getName()
-    ).toMatchInlineSnapshot('"bcCCNc"');
```

Comment:
I believe this test was wrong before. We are using the `'../../test/utils'` and it should mock the `generateAlphabeticName`